crates/flipperzero/src/gpio/i2c.rs

```diff
@@ -309,13 +309,410 @@ impl BusHandle {
             Err(Error::TransferFailed)
         }
     }
+
+    /// Reads data from `device` and writes it to the `data` buffer.
+    ///
+    /// `timeout` is in milliseconds.
```

Comment:
No longer relevant because you use `Duration`.

Reply:
Yes please. That would be appreciated (I didn't see that it was also on the other `Duration` methods).

Reply:
Done